features/48-user-profile-discovery/field-based-acl.md

```diff
@@ -197,6 +235,61 @@ The sample request is shown as followed:
 }
 ```
 
+## Record Partial Update
+
+To support record partial update, warnings are added to the response
+interface of `record:update` action to indicate the rejected fields on a
+**non-atomic** save action.
+
+The following is a sample response of a record partial update:
+
+```js
+{
+  "result": [
+    {
+      // record that saved successfully
+      "_access": [
+        { "level": "read", "public": true }
+      ],
+      "_created_at": "2017-06-19T06:06:16.734626Z",
+      "_created_by": "3c25ddff-b6d4-4d1d-8e67-45c3d82c90f5",
+      "_id": "note/1",
+      "_ownerID": "3c25ddff-b6d4-4d1d-8e67-45c3d82c90f5",
+      "_type": "record",
+      "_updated_at": "2017-06-19T06:33:33.081303Z",
+      "_updated_by": "3c25ddff-b6d4-4d1d-8e67-45c3d82c90f5",
+      "content": "hello world",
+      "tags": ["important", "must-read"]
+    },
+    {
+      // record that saved partially
+      "_access": [
+        { "level": "read", "public": true }
+      ],
+      "_created_at": "2017-06-19T06:06:30.125993Z",
+      "_created_by": "3c25ddff-b6d4-4d1d-8e67-45c3d82c90f5",
+      "_id": "note/2",
+      "_ownerID": "3c25ddff-b6d4-4d1d-8e67-45c3d82c90f5",
+      // list of fields rejected from update
+      "_rejected_fields": ["tags"],
```

Comment:
The naming `reject` does not tell any about access control?

We can keep `reject` and make it array of object. It is more extensiable.
`"_rejected_fields": [{"key": "tags", "reason":"access denied"}]`

Or we can make it more explicit by naming is `_access_denied_fields`, so we are more confident that it will only need an array of keys.